tools/clang/include/clang/SPIRV/SpirvContext.h

```diff
@@ -165,6 +165,20 @@ class SpirvContext {
                                            SpirvConstant *size,
                                            uint32_t encoding);
 
+  SpirvDebugInstruction *
+  getDebugTypeMember(llvm::StringRef name, const SpirvType *type,
```

Comment:
I think the return type of this function should be `SpirvDebugTypeMember *` ?

Reply:
actually, never mind about this one. The member can be a member variable or it can be a member function.